### PR TITLE
merge head

### PR DESCRIPTION
**Pull request by 7ez**

### CODE REVIEW

**Comment by 7ez** [comment]
@RealistikDash could you fix these for me

**Comment by RealistikDash** [comment]
Ideally we don't want those.

**Comment by 7ez** [comment]
> Ideally we don't want those.

Why not

**Comment by RealistikDash** [comment]
@tsunyoku @cmyui wanna be code owners!?!?

**Comment by 7ez** [comment]
> @tsunyoku @cmyui wanna be code owners!?!?

you can literally merge it and change codeowners afterwards.

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,6 @@
+* @tsunyoku @cmyui
+
+# Infrastructure
+/.github/* @cmyui
+/tf/* @cmyui
+/chart/* @cmyui
```
Comment (cmyui) [inline CODEOWNERS:6]:
```suggestion
* @RealistikDash @lenforiee
```

Comment (cmyui) [reply CODEOWNERS:6]: might as well rebase anyways given https://github.com/RealistikOsu/akatsuki-pp-rs/commit/b78c986a82f158cb26f05662e55a00eac6641c06

**Review by RealistikDash — APPROVED** [review]

**Pull request closed without merging**